### PR TITLE
CLC-5827 Front end for OEC control panel

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/CLC-5827

Having the API serve up task descriptions helps make the whole thing a little less opaque. Note that ExportTask is advertised as a validation task, while PublishTask is advertised as an export task. https://jira.ets.berkeley.edu/jira/browse/CLC-5833 will bring these into sync.

### CODE REVIEW

**Comment by johncrossman** [comment]
This looks muy good.

File: src/assets/templates/oec.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<div class="cc-oec" data-cc-spinner-directive>
+  <div data-ng-if="!displayError">
+    <h1 class="cc-heading-page-title">OEC Control Panel</h1>
+    <div class="cc-oec-content medium-12 columns">
+
+      <form data-ng-if="!taskInProgress" data-ng-submit="runOecTask()">
+
+        <div class="row">
+          <div class="small-3 columns">
+            <label class="cc-oec-label">Task to run:</label>
```
Comment (christianv) [inline src/assets/templates/oec.html]: @paulkerschen we need to add the `for` attribute here so when you click the label, it opens the dropdown.

File: src/assets/templates/oec.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+              <select data-ng-model="taskParameters.selectedTask" data-ng-options="task.friendlyName for task in oecTasks">
+                <option value="">Choose a task</option>
+              </select>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="small-3 columns">
+            <label class="cc-oec-label">Term:</label>
```
Comment (christianv) [inline src/assets/templates/oec.html]: @paulkerschen same as above, add `for`

File: src/assets/templates/oec.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+          <div class="small-9 columns">
+            <div class="cc-select">
+              <select data-ng-options="term for term in oecTerms" data-ng-model="taskParameters.options.term"></select>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="small-3 columns">
+            <label class="cc-oec-label">Department(s):</label>
```
Comment (christianv) [inline src/assets/templates/oec.html]: @paulkerschen same as above, add `for`

**Pull request merged**